content/v1.10-docs/manifest.json

```diff
@@ -5,179 +5,179 @@
       "routes": [
         {
           "title": "Introduction",
-          "path": "/docs/README.md"
+          "path": "/v1.10-docs/README.md"
         },
         {
             "title": "Getting Started",
-            "path": "/docs/getting-started/README.md"
+            "path": "/v1.10-docs/getting-started/README.md"
         },
         {
           "title": "Installation",
           "routes": [
             {
               "title": "Introduction",
-              "path": "/docs/installation/README.md"
+              "path": "/v1.10-docs/installation/README.md"
             },
             {
               "title": "Supported Releases",
-              "path": "/docs/installation/supported-releases.md"
+              "path": "/v1.10-docs/installation/supported-releases.md"
             },
             {
               "title": "Cloud Compatibility",
-              "path": "/docs/installation/compatibility.md"
+              "path": "/v1.10-docs/installation/compatibility.md"
             },
             {
               "title": "Kubectl apply",
-              "path": "/docs/installation/kubectl.md"
+              "path": "/v1.10-docs/installation/kubectl.md"
             },
             {
               "title": "Kubectl plugin",
-              "path": "/docs/installation/kubectl-plugin.md"
+              "path": "/v1.10-docs/installation/kubectl-plugin.md"
             },
             {
               "title": "cmctl",
-              "path": "/docs/installation/cmctl.md"
+              "path": "/v1.10-docs/installation/cmctl.md"
             },
             {
               "title": "Helm",
-              "path": "/docs/installation/helm.md"
+              "path": "/v1.10-docs/installation/helm.md"
             },
             {
               "title": "OLM",
-              "path": "/docs/installation/operator-lifecycle-manager.md"
+              "path": "/v1.10-docs/installation/operator-lifecycle-manager.md"
             },
             {
               "title": "Other tools",
-              "path": "/docs/installation/other-tools.md"
+              "path": "/v1.10-docs/installation/other-tools.md"
             },
             {
               "title": "Verifying",
-              "path": "/docs/installation/verify.md"
+              "path": "/v1.10-docs/installation/verify.md"
             },
             {
               "title": "Upgrading",
               "routes": [
                 {
                   "title": "Introduction",
-                  "path": "/docs/installation/upgrading/README.md"
+                  "path": "/v1.10-docs/installation/upgrading/README.md"
                 },
                 {
                   "title": "Notes on Ingress Class Compatibility",
```

Comment:
suggestion: sorry, I missed this first time through. There are actually a bunch of docs which we don't include for versions, including "upgrading", "contributing" and release notes.

E.g. in https://cert-manager.io/v1.8-docs and https://cert-manager.io/v1.9-docs

Removing those would also reduce the size of this PR by a decent amount too!

This is another of those things which should be automated but isn't :sob: